README.rst

```diff
@@ -34,6 +34,61 @@ One Time Setup
   # Here's how you might do that if you have virtualenvwrapper setup.
   mkvirtualenv -p python3.8 learning-assistant
 
+In your ``private.txt`` requirements file in edx-platform, add:
```

Comment:
Can you specify `requirements/edx/private.txt` to be clear which file this is, please?